Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -363,17 +371,28 @@ int testModel(const char *data_path, const char *model) {
 
 fail_info_release:
   ml_tensors_info_destroy(in_info);
-
+  originStatus = status;
   status = ml_pipeline_stop(pipe);
+  NN_STATUS_RETURN();
```

Comment:
This exits without `sink_unregister`, `src_release_handle` and `destroy_pipeline` if `pipeline_stop` fails. This will leak memory. Please correct this.

Reply:
All right.
But let say something went wrong so exit to fail_info_release and ml_pipeline_stop also failed.
Then which status should I return?

Reply:
The original status of failure, which led to the exit to fail_info_release.

Reply:
Done